scripts/apps/search/directives/SearchParameters.js

```diff
@@ -403,6 +404,10 @@ export function SearchParameters($location, asset, tags, metadata, common, desks
                     delete scope.fields[type];
                 }
             };
+
+            scope.getTemplate = (providerType) => `search-panel-${providerType}.html`;
+
+            scope.hasTemplate = (providerType) => providerType && !!$templateCache.get(scope.getTemplate(providerType));
```

Comment:
Could you apply the same changes here as well? I mean comparing explicitly and not using type coercion

Reply:
:ok:

Reply:
Here's a fix for type coercion `!!$templateCache.get(scope.getTemplate(providerType))` -> `$templateCache.get(scope.getTemplate(providerType)) != null`

Thanks :)